hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataReader.java

```diff
@@ -114,8 +115,8 @@ public OmMetadataReader(KeyManager keyManager,
         authorizer.setBucketManager(bucketManager);
         authorizer.setKeyManager(keyManager);
         authorizer.setPrefixManager(prefixManager);
-        authorizer.setOzoneAdmins(
-            new OzoneAdmins(ozoneManager.getOmAdminUsernames()));
+        authorizer.setOzoneAdmins(ozoneManager.getOmAdmins());
+        authorizer.setOzoneSuperReadAdmins(getOmSuperReadAdmins(configuration));
```

Comment:
Could you please change all SuperRead to ReadOnly in this one and all the following new codes? We should keep the naming consistent.

Reply:
okey, I will change